Makefile

```diff
@@ -1,5 +1,11 @@
+.PHONY: requirements
+ci_requirements:
+	python3 -m pip install uv
+	python3 -m uv pip install --system -r requirements/development.txt
+
 .PHONY: requirements
 requirements:
+	python3 -m pip install uv
 	python3 -m pip install -r requirements/development.txt
```

Comment:
This line can be changed as well

Reply:
I have it this way to avoid making assumptions about the user's virtual environment setup. It's fine for our CI target, though.